kv/kvclient/kvcoord.TestRangeFeedMetricsManagement: TestRangeFeedMetricsManagement failed

kv/kvclient/kvcoord.TestRangeFeedMetricsManagement.TestRangeFeedMetricsManagement [failed](https://mesolite.cluster.engflow.com/invocations/default/faad93fb-e7b3-4597-bd2d-c3baca91d155?testReportRun=1&testReportShard=13&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZjbGllbnQva3Zjb29yZDprdmNvb3JkX3Rlc3Q=) on master @ [8e56ad9c46e544c2ef368e02322d949a03723891](https://github.com/cockroachdb/cockroach/commits/8e56ad9c46e544c2ef368e02322d949a03723891):


```
=== RUN   TestRangeFeedMetricsManagement
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestRangeFeedMetricsManagement2928220601
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:141: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    test_server_shim.go:594: DRPC is enabled (override by TestingGlobalDRPCOption)
    dist_sender_rangefeed_test.go:561: condition failed to evaluate within 10s: from dist_sender_rangefeed_test.go:563: waiting for frontier advance: [/Table/104/1{-/300}@0,0] [/Table/104/1/{300-500}@1762260328.598825369,0] [/Table/104/1/{500-800}@1762260336.937648510,0] [/Table/104/{1/800-2}@0,0]
    panic.go:615: -- test log scope end --
test logs left over in: outputs.zip/logTestRangeFeedMetricsManagement2928220601
--- FAIL: TestRangeFeedMetricsManagement (14.55s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=13</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRangeFeedMetricsManagement.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-56227

--- a/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed_test.go
+++ b/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed_test.go
@@ -558,12 +558,12 @@ func TestRangeFeedMetricsManagement(t *testing.T) {
 
 	// Wait for the test frontier to advance.  Once it advances,
 	// we know the rangefeed is started, all ranges are running (even if some of them are blocked).
-	testutils.SucceedsWithin(t, func() error {
+	testutils.SucceedsSoon(t, func() error {
 		if frontier.Frontier().IsEmpty() {
 			return errors.Newf("waiting for frontier advance: %s", frontier.String())
 		}
 		return nil
-	}, 10*time.Second)
+	})
 
 	// At this point, we know the rangefeed for all ranges are running.
 	require.EqualValues(t, numRanges, metrics.RangefeedRanges.Value(), frontier.String())

